satpy/readers/fci_l1c_nc.py

```diff
@@ -233,6 +234,7 @@ def __init__(self, filename, filename_info, filetype_info):
         else:
             self.is_iqt = False
 
+        self.clip_negative_radiances = clip_negative_radiances
```

Comment:
Don't you need to access the `satpy.config` object to get the default for this?

Reply:
Hmm, yes I should.  Will add that!

Reply:
Added and tested now.